python_package/run_brainflow.py

```diff
@@ -0,0 +1,17 @@
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, BoardIds
+
+BoardShim.enable_dev_board_logger()
+
+
+params = BrainFlowInputParams()
+params.mac_address = "00:13:43:A1:84:EE"
```

Comment:
if you want to use it as an example it makes sense to add mac address as an optional argument, put it in examples/tests folder and rename to smth like explore_plus_8_channels_test.py